.env

```diff
@@ -0,0 +1,2 @@
+NEXT_PUBLIC_BASE_PATH=https://api.nytimes.com
```

Comment:
please rename this file to `.env.local`. 
We should not have this file in github